app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(
+      text: params[:text],
+      post_id: params[:post_id]
+    )
+    @comment.user_id = current_user.id
```

Comment:
**OPTIONAL**
- Well done creating the `create` method and the `@comment` instance variable correctly. However, using the `private` keyword in the controllers to create the `params` method is very useful for the application's security and performance. Please learn more [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md).
```ruby
def create
  @comment = current_user.comments.new(comment_params)
  @comment.post_id =  params[:post_id]
  ...
end
private
def comment_params
  params.require(:comment).permit(:text)
end
```